opencti-platform/opencti-graphql/src/modules/case/case-rfi/case-rfi.graphql

```diff
@@ -96,6 +96,7 @@ type CaseRfi implements BasicObject & StixObject & StixCoreObject & StixDomainOb
   objectAssignee: AssigneeConnection
   avatar: OpenCtiFile
   # Container
+  is_from_relation_inferred: Boolean
```

Comment:
Not sure about the placement of this. We need to check if it could be more interesting to put the information in the edge type